roles/openshift_logging/tasks/delete_logging.yaml

```diff
@@ -107,6 +107,25 @@
     - logging-fluentd
     - logging-mux
 
+# remove annotations added by logging
+- oc_obj:
```

Comment:
is this step necessary if we already have the names of the ops projects?

Reply:
I believe, similar to the annotation task, we list the projects and get back those that actually exist in context of those we  expect to be there.  We then use that result to further remove the annotation.  We can remove this step if you feel removing the annotation and ignoring failure should not actual exist is acceptable